tests/storybook.ts

```diff
@@ -4,9 +4,14 @@ import { RunOptions } from '../types'
 export async function test(options: RunOptions) {
 	await runInRepo({
 		...options,
-		repo: 'storybookjs/builder-vite',
-		branch: 'main',
-		build: 'prepublish',
-		test: ['build-examples', 'test-ci']
+		repo: 'storybookjs/repro-templates-temp',
+		dir: 'storybook7',
+		packageDir: 'react-vite/default-js/after-storybook',
+		branch: 'next',
+		build: ['yarn install', 'build-storybook'],
+		test: [
+			'yarn add --dev @storybook/test-runner http-server concurrently',
+			'npx concurrently -k "http-server storybook-static --port 6006 --silent" "npx wait-on http://localhost:6006 && npx test-storybook"'
```

Comment:
is it possible to not use concurrently here? ecosystem ci calls this in a subprocess but not sure it behaves when cancelled

Reply:
I need to be able to start up a web server, wait for it to start, run my tests, and then tear everything down when the tests are finished.  Do you know of a better way to accomplish that?

Reply:
no clue with storybook's own testrunner, sorry. vitest and playwright have utilities for that